app/views/rubric_criteria/_criterion_editor.html.erb

```diff
@@ -1,8 +1,15 @@
 <div class='float-right'>
+
+  <% released = !criterion.results_unreleased? %>
```

Comment:
This is correct but a bit inefficient, since it'll be repeated for each criterion (and rubric level). Instead, try to figure out how the existing flash message "Cannot update criteria for an assignment with released marks." is being displayed, presumably conditioned on some boolean. That same boolean can be passed into this partial as a `released` variable, and you can use that directly. Same with the rubric levels.

On a related note, please modify the "Cannot update criteria for an assignment with released marks." so that it appears as an "info" rather than "error", since technically there's no error, we're just letting instructors know they can't change anything.